RocketChat/Rocket.Chat
# feat: add new feature Mute unmute feature added to RoomMenu Dropdown

## Proposed changes (including videos or screenshots)

I have implemented a mute/unmute feature in the dropdown of the `RoomMenu.tsx` file, addressing the issue where users could not mute or unmute other users directly from the room menu. A new button for muting and unmuting users has been added to the dropdown menu for every channel.

Video demonstration:  
[![Loom Video](https://www.loom.com/share/8d590f4655054081b8b95a5b5325be90?sid=4c6c27a8-55a7-4306-a36e-7373a754f77f)](https://www.loom.com/share/8d590f4655054081b8b95a5b5325be90?sid=4c6c27a8-55a7-4306-a36e-7373a754f77f)

![Screenshot from 2024-09-17 00-10-16](https://github.com/user-attachments/assets/408c1aca-8b61-4756-8d31-43287d9a2335)


## Issue(s)

Closes #11954

## Steps to test or reproduce

1. Go to any channel in the application.
2. Open the dropdown menu in the `RoomMenu.tsx` file.
3. You should now see the new mute/unmute button.
4. Use the button to mute or unmute users in the channel.

## Further comments

This feature allows users to mute or unmute others directly from the dropdown menu within the channel. The solution was chosen to enhance user control within chat rooms, making it more intuitive. No significant alternatives were considered as this directly resolves the identified issue efficiently.
